xml/System.IO.Pipes/AnonymousPipeServerStreamAcl.xml

```diff
@@ -40,13 +40,25 @@
         <Parameter Name="pipeSecurity" Type="System.IO.Pipes.PipeSecurity" />
       </Parameters>
       <Docs>
-        <param name="direction">To be added.</param>
-        <param name="inheritability">To be added.</param>
-        <param name="bufferSize">To be added.</param>
-        <param name="pipeSecurity">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="direction">One of the enumeration values that determines the direction of the pipe. Anonymous pipes can only be in one direction, so direction cannot be set to <see cref="F:System.IO.Pipes.PipeDirection.InOut" />.</param>
+        <param name="inheritability">One of the enumeration values that determines whether the underlying handle can be inherited by child processes.</param>
+        <param name="bufferSize">The size of the buffer. This value must be greater than or equal to 0.</param>
+        <param name="pipeSecurity">An object that determines the access control and audit security for the pipe.</param>
+        <summary>Creates a new instance of the <see cref="T:System.IO.Pipes.AnonymousPipeServerStream" /> class with the specified pipe direction, inheritability mode, buffer size, and pipe security.</summary>
+        <returns>A new anonymous pipe server stream instance.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Setting `pipeSecurity` to `null` is equivalent to calling the `System.IO.Pipes.AnonymousPipeServerStream(System.IO.Pipes.PipeDirection direction, System.IO.HandleInheritability inheritability, int bufferSize)` constructor directly.
+
+          ]]></format>
+        </remarks>
+        <exception cref="T:System.NotSupportedException">
+          <paramref name="direction" /> is <see cref="F:System.IO.Pipes.PipeDirection.InOut" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="inheritability" /> is not set to a valid <see cref="T:System.IO.HandleInheritability" /> enum value.</exception>
```

Comment:
You mentioned earlier that bufferSize value must be greater than or equal to 0. Shouldn't be there an exception for it?

Reply:
That's correct:

```suggestion
          <paramref name="inheritability" /> is not set to a valid <see cref="T:System.IO.HandleInheritability" /> enum value.

 -or-  
  
<paramref name="bufferSize" /> is less than 0.</exception></exception>
```